src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -180,6 +189,16 @@ FullyConnected::FullyConnected(const std::shared_ptr<ngraph::Node>& op, const Gr
         minSparseRate = context->getConfig().fcSparseWeiDecompressionRate;
 
     expectedBiasDims = {getInputShapeAtPort(WEIGHTS_ID).getStaticDims()[0]};
+
+#ifdef OV_CPU_WITH_LLMDNN
+    const std::string& pkey = ov::PrimitivesPriority::get_type_info_static();
+    const auto& info = op->get_rt_info();
+    if (info.count(pkey)) {
+        auto primitives_priority_attr = info.at(pkey).as<ov::PrimitivesPriority>().value;
+        if (primitives_priority_attr.find("gemm_llmdnn") == std::string::npos)
+            stateLLMFc = State_NotUse;
```

Comment:
Why do we need this logistics here that to get a dependency to rt_info? And who will put this "gemm_llmdnn" info?

Reply:
User/testcase can add rtinfo to select prefer kernel. The logic is updated to more robust method.